LML.NPOManagement.Common/Model/ServiceStatusCode.cs

```diff
@@ -11,6 +11,7 @@ public enum ServiceStatusCode
         Conflict = 7,
         BadRequest = 8,
         InternalServerError = 9,
-        Unauthorized = 10
+        Unauthorized = 10,
+        Acepted = 11
```

Comment:
Typo in "Accepted", also please explain the use cases of introducing additional status code

Reply:
Accepted status is used to identify that user is already active and there is no need to send another verification email to user

Reply:
i have changed now if user is already active it just send status code 200 without sending new email